upgrading/topics/keycloak/changes.adoc

```diff
@@ -230,7 +230,7 @@ logs in.
 
 ==== Non-standard token introspection endpoint removed
 
-In previous versions, Keycloak advertized two introspection endpoints: `token_introspection_endpoint` and `introspection_endpoint`.
+In previous versions, Keycloak advertised two introspection endpoints: `token_introspection_endpoint` and `introspection_endpoint`.
```

Comment:
Advertized is the US English spelling, which is the one we use.  Advertised is the British English spelling.

Reply:
@andymunro, I think it is advertised even in US english. See https://github.com/keycloak/keycloak-documentation/pull/1625#discussion_r928106385.

Reply:
Okay, thanks again.